spec/models/target_spec.rb

```diff
@@ -0,0 +1,38 @@
+# == Schema Information
+#
+# Table name: targets
+#
+#  id         :bigint           not null, primary key
+#  title      :string
+#  radius     :decimal(, )
+#  latitude   :decimal(, )
+#  longitude  :decimal(, )
+#  topic_id   :bigint           not null
+#  user_id    :bigint           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+require 'rails_helper'
+
+describe Target, type: :model do
+
+  let(:user) { create(:user) }
+
+  let(:topic_list) { create_list(:topic, 5) }   
+
+  let!(:target_list) { create_list(:target, 3, user: user, topic: topic_list.first) }
+
+  let(:target) { Target.create(title: 'This cannot be created', radius: 5, latitude: 6.244203, longitude: -75.581215, topic_id: topic_list.first.id, user_id: user.id) }
```

Comment:
```suggestion
  let(:target) { create(:target, user_id: user.id) }
```
Please, try using factories with only the attributes needed, like in this case, the user this target belongs to.

Reply:
In this case, it will be necessary for custom model validations

Reply:
If you need you can set attributes in the factory too!
Like:
`let(:target) { create(:target, title: 'This cannot be created' , radius: 5 , ....`